library_builders.sh

```diff
@@ -287,13 +342,12 @@ function build_hdf5 {
         HOST_ARG="--host=aarch64-apple-darwin"
 
         curl -sLo osx_cross_configure.patch \
-            https://raw.githubusercontent.com/h5py/h5py/fcaca1d1b81d25c0d83b11d5bdf497469b5980e9/ci/osx_cross_configure.patch
+            https://raw.githubusercontent.com/conda-forge/hdf5-feedstock/ca7b3b40d58cb949638ead75c9527b3fd1112a3f/recipe/patches/osx_cross_configure.patch
```

Comment:
Ok, the only problem left here is that these two patches for H5detect to use the host native version do not seem to work yet (macOS arm64).

Probably needs to wait until after my vacation.

Reply:
We will stay with HDF5 1.12.2 for now on Unix to move on.